.circleci/config.yml

```diff
@@ -102,3 +105,6 @@ workflows:
           context: QualityTime
       - unittest_docs:
           context: QualityTime
+      - docker/hadolint:
+          context: QualityTime
+          dockerfiles: components/collector/Dockerfile:components/database/Dockerfile:components/external_server/Dockerfile:components/frontend/Dockerfile:components/internal_server/Dockerfile:components/ldap/Dockerfile:components/notifier/Dockerfile:components/proxy/Dockerfile:components/renderer/Dockerfile:components/testdata/Dockerfile
```

Comment:
Is it possible to expand this to multiple lines for readablity?

Reply:
I don't know, I'll give it a try.

Reply:
Yes, line can be broken up:

```yaml
workflows:
  build_test:
    jobs:
      - docker/hadolint:
          dockerfiles: "components/collector/Dockerfile:\
            components/database/Dockerfile:\
            components/external_server/Dockerfile:\
            components/frontend/Dockerfile:\
            components/internal_server/Dockerfile:\
            components/ldap/Dockerfile:\
            components/notifier/Dockerfile:\
            components/proxy/Dockerfile:\
            components/renderer/Dockerfile:\
            components/testdata/Dockerfile"
```